### PR TITLE
Refactor land.js

### PR DESCRIPTION
**Pull request by bumjung**

### CODE REVIEW

File: modules/simulator.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+$(function()
+{
+  var canvas = $("#canvas")[0];
+  var ctx = canvas.getContext('2d');
```
Comment (bumjung) [inline modules/simulator.js:4]: probably can move this into step()

Comment (insoochoo) [inline modules/simulator.js:4]: moving this into step() means it will be repeated which we dont want

**Comment by insoochoo** [comment]
Looks good to me :+1: 

lets decide to always do brackets on a new line
i.e. 
if (...)
{

}

File: modules/simulator.js (hunk 1 above)
Comment (bumjung) [inline modules/simulator.js]: yes, I agree - this is for slack integration test

**Comment by bumjung** [comment]
this is a comment

File: modules/simulator.js (hunk 1 above)
Comment (bumjung) [inline modules/simulator.js]: this is a comment on a commit

Comment (bumjung) [inline modules/simulator.js:4]: good point :+1:

**Pull request merged**